### DISCUSSION TITLE
How to rename a struct?

### DISCUSSION THREAD

**Discussion by xuxiaocheng0201** [fzyzcjy/flutter_rust_bridge#2229] [Q&A] [answered]
I read the docs, both [fields](https://cjycode.com/flutter_rust_bridge/guides/types/translatable/detailed/struct#rename-fields) and [functions](https://cjycode.com/flutter_rust_bridge/guides/functions/renaming) can be renamed by add `#[frb(name = "NewName")]`.But I couldn't find how to rename a method or a struct. I tried the same attribute `name`, it not worked.

**Comment by fzyzcjy** [comment, marked as answer]
Hmm, if the `name = ...` does not work, then it is probably because it has not been implemented. Feel free to PR for this! Alternatively, I may work on it later but may not guarantee very soon since this is not a core issue.
